Hardware/Star64.md

```diff
@@ -0,0 +1,34 @@
+---
+riscv_hardware: true
+cmake_plat: star64
```

Comment:
Maybe we should call this `pine-star64` to make the name a bit more unique. Or `jh7110-star64`, because there are more board with the JH7110 SOC. I have a branch for the VisionFive, where the v2 board version will also use this SOC.

Reply:
I can change it to something like `pine64-star64` but it looks like other boards by Pine64 just list the board name. Is it expected for another board to exist also called Star64 with a different SoC? I thought that just having Star64 should be disambiguous enough but maybe not.

Reply:
So the name on the [Pine64 store](https://pine64.com/product/star64-model-a-4gb-single-board-computer/) is "STAR64 Model-A" so might be worth changing this to something like `star64_a`